ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -80,10 +80,97 @@ <h1 class="card-title">Review complete!</h1>
         </div>
     </div>
 </div>
+<!-- Modal -->
+<div class="modal fade" id="confirm-modal" tabindex="-1" role="dialog" aria-labelledby="confirm-modal-label" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="confirm-modal-label">Are you sure?</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <span id="confirm-message"><b>Are you sure you want to request teacher review?</b><br>
+                    Please listen back to your audio recording to ensure that there was a grading or 
+                    transcription error before requesting review.<br>
+                    If you have already done so, continue. 
```

Comment:
Eventually, we should try to force the user to do this. Doesn't matter for this pr, but we should make an issue for it

Reply:
sounds good! Ill make an issue rn! made issue #245